net/atlas-sw-probe/Makefile

```
@@ -30,7 +30,7 @@ define Package/atlas-sw-probe
   CATEGORY:=Network
   TITLE:=RIPE Atlas software probe
   URL:=https://atlas.ripe.net/about/probes/
-  DEPENDS:=+atlas-probe +PACKAGE_dropbear:dropbearconvert
+  DEPENDS:=+atlas-probe openssh-keygen
```

Just noticed this, I think you should do:
```suggestion
  DEPENDS:=+atlas-probe +openssh-keygen
```
As per https://openwrt.org/docs/guide-developer/packages#dependency_types